### PR TITLE
Fix: updated Navbar Items btn bg

### PR DESCRIPTION
**Pull request by j4nlksh**
What does this PR do?

This pull request updates the background of the navbar items in the Next.js project using Tailwind CSS, ensuring improved visual consistency and a more appealing user interface.

Fixes #1314
Issue #1314

This Particular PR is fix for **Issue #1314** From the Repo - **responsively-org/responsively-app** - as per needed!

## Mandatory Tasks (DO NOT REMOVE)

- [ ] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by j4nlksh** [comment]
@esprush & @themohammadsa - Guy's Please Give it a Merge! 
Majorly, I've fixed what needed to be, if something don't go right - please lemme know!

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "focus-visible": "^5.2.0",
     "gray-matter": "^4.0.3",
     "javascript-time-ago": "^2.5.9",
-    "next": "13.3.0",
+    "next": "^14.2.14",
```
Comment (violetadev) [inline package.json:24]: please remove any upgrades, your pr should not be changing this

File: next-env.d.ts [hunk 2]
```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
```
Comment (violetadev) [inline next-env.d.ts:5]: please remove this, your pr should not be modifying this file

File: package.json (hunk 1 above)
Comment (j4nlksh) [reply package.json]: Is it ok now?

Comment (violetadev) [reply package.json]: I still see some unwanted changes on the other files.
I think the best way to fix all the extra files that are not needed is to copy paste your one liner change in a notepad, then do a reset hard of the whole branch to main, and then readd your fix and commit only this file with `git add src/components/NavLink.jsx`

Comment (j4nlksh) [reply package.json]: - @violetadev , Give it a check nowww?!

**Review by j4nlksh — COMMENTED** [review]
done

**Pull request merged**